vitest.config.js

```
@@ -0,0 +1,20 @@
+import { defineConfig } from "vitest/config";
+
+export default defineConfig({
+  test: {
+    globals: true,
+    environment: "node",
```

ESLint

[error] Expected "environment" to come before "globals".

(perfectionist/sort-objects)